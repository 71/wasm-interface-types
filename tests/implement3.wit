;; parse-fail

(module
  (import "" "" (func))
  (import "" "" (func))
  (@interface implement (import "" "") (func 1)))
