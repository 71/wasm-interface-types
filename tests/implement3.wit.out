import of `` from `` is ambiguous since it's listed twice in the core module
     --> ../../tests/implement3.wit:6:15
      |
    6 |   (@interface implement (import "" "") (func 1)))
      |               ^
